### PR TITLE
LPS-162054 Custom Facet Ordering

### PR DESCRIPTION
**Pull request by felipelorenz**
https://issues.liferay.com/browse/LPS-162054

### CODE REVIEW

**Comment by felipelorenz** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/custom/facet/display/context/CustomFacetDisplayContextTest.java [hunk 1]
```diff
@@ -179,11 +182,209 @@ public void testOneTermWithPreviousSelection() throws Exception {
...
+
+		nameFrequencyString = _buildNameFrequencyString(bucketDisplayContexts);
+
+		Assert.assertEquals(
+			bucketDisplayContexts.toString(),
+			"charlie:6|bravo:5|delta:5|alpha:4", nameFrequencyString);
+	}
+
+	@Test
+	public void testOrderByTermValueAscending() throws Exception {
```
Comment (gustavolimav) [inline modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/custom/facet/display/context/CustomFacetDisplayContextTest.java:272]: Why are you testing order by Value Ascending and Value Descending if the user cannot order this facet by any of those?

**Review by gustavolimav — CHANGES_REQUESTED** [review]
Remove testOrderByTermValueAscending  and testOrderByTermValueDescending

\cc @felipelorenz

**Comment by gustavolimav** [comment]
remember to put the ticket in the description

File: modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/custom/facet/display/context/CustomFacetDisplayContextTest.java (hunk 1 above)
Comment (BryanEngler) [reply modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/custom/facet/display/context/CustomFacetDisplayContextTest.java]: @gustavolimav why cant we order by value? i dont think theres any reason why we shouldnt be able to sort on value... let me know if im missing something, but i think those options need to be added to the config UI

**Comment by felipelorenz** [comment]
ci:test:sf

File: modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/custom/facet/display/context/CustomFacetDisplayContextTest.java (hunk 1 above)
Comment (felipelorenz) [reply modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/custom/facet/display/context/CustomFacetDisplayContextTest.java]: @BryanEngler In this story we only have two options to ordering, by frequency ascending and frequency descending. We dont have the value, like the others.

**Comment by felipelorenz** [comment]
ci:test:search

**Comment by felipelorenz** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/custom/facet/display/context/CustomFacetDisplayContextTest.java (hunk 1 above)
Comment (BryanEngler) [reply modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/custom/facet/display/context/CustomFacetDisplayContextTest.java]: @felipelorenz ya i think the story is wrong. Could check with @lipusz, but i think all 4 options should be there

**Comment by felipelorenz** [comment]
ci:test:sf

**Comment by felipelorenz** [comment]
ci:test:search

**Comment by felipelorenz** [comment]
ci:test:relevant

**Comment by joshuacords** [comment]
Hi @felipelorenz, could you simplify the tests here like was done in https://github.com/liferay-search/liferay-portal/pull/705?

**Pull request closed without merging**